torch/csrc/distributed/rpc/init.cpp

```diff
@@ -214,6 +214,78 @@ PyObject* rpc_init(PyObject* /* unused */) {
                   If the current node is the owner, returns a reference to the
                   local value. Otherwise, throws an exception.
               )")
+          .def(
+              "rpc_sync",
+              [&](PyRRef& self) {
+                  return self.createRRefProxy(self, RRefProxyType::RPC_SYNC);
+              },
+              py::call_guard<py::gil_scoped_release>(),
+              R"(
+                  Create a helper proxy to easily launch an ``rpc_sync`` using
+                  the owner of the RRef as the destination to run functions on
+                  the object referenced by this RRef. More specifically, 
+                  ``rref.rpc_sync().func_name(*args, **kwargs)`` is the same as
+                  the following:
+
+                  >>> def run(rref, func_name, args, kwargs):
+                  >>>   return getattr(rref.local_value(), func_name)(*args, **kwargs)
+                  >>>
+                  >>> rpc.rpc_sync(rref.owner(), run, args=(rref, func_name, args, kwargs))
+
+                  Example::
+                      >>> from torch.distributed import rpc
+                      >>> rref = rpc.remote("worker1", torch.add, args=(torch.zeros(2, 2), 1))
+                      >>> rref.rpc_sync().size()  # returns torch.Size([2, 2])
+                      >>> rref.rpc_sync().view(1, 4)  # returns tensor([[1., 1., 1., 1.]])
+              )")
+          .def(
+              "rpc_async",
+              [&](PyRRef& self) {
+                  return self.createRRefProxy(self, RRefProxyType::RPC_ASYNC);
+              },
+              py::call_guard<py::gil_scoped_release>(),
+              R"(
+                  Create a helper proxy to easily launch an ``rpc_async`` using
+                  the owner of the RRef as the destination to run functions on
+                  the object referenced by this RRef. More specifically, 
+                  ``rref.rpc_async().func_name(*args, **kwargs)`` is the same as
+                  the following:
+
+                  >>> def run(rref, func_name, args, kwargs):
+                  >>>   return getattr(rref.local_value(), func_name)(*args, **kwargs)
+                  >>>
+                  >>> rpc.rpc_async(rref.owner(), run, args=(rref, func_name, args, kwargs))
+
+                  Example::
+                      >>> from torch.distributed import rpc
+                      >>> rref = rpc.remote("worker1", torch.add, args=(torch.zeros(2, 2), 1))
+                      >>> rref.rpc_async().wait().size()  # returns torch.Size([2, 2])
+                      >>> rref.rpc_async().wait().view(1, 4)  # returns tensor([[1., 1., 1., 1.]])
```

Comment:
should this be rref.rpc_async().view(1, 4).wait()

Reply:
Ahh, yes, thanks for the catch!!